### ISSUE TITLE
settings

### ISSUE THREAD

**Issue by zartan9** [haroldo-ok/BlocklyVN8bit#17]
hi, when loading from blocklyvn8bit to test, the c64 runs (on warp) speedily. but if I drag d64 into emulator it shows black and white lines (on warp), how can i set it to test the same way? Im adding delays in the game and on warp it seems its quick no matter what #
i put in? sorry if im not clear, but when I dble click the test c64 batch file, works nice,
but d64 is slow...

**Comment by haroldo-ok** [comment]
Weird, I think the batch file that calls the emulator is using the default settings, so there should be no difference;  gotta take a look at them later.

**Comment by haroldo-ok** [comment]
Checking the code, it seems it is really using the default settings:

```batch
(...)
echo DONE
pause

cd "utils\emulators\WinVICE-2.4"
x64.exe "..\..\..\build\test-c64.d64"
```

As you can see, `x64.exe "..\..\..\build\test-c64.d64"` is calling the emulator simply passing the disk as a parameter, dragging and dropping should give the same results in theory.

**Comment by zartan9** [comment]
i see! if i ever get to makin a game (and try to sell it!) should i distribute the game batch file  (with the d64) to make sure their emulators open right?

**Comment by haroldo-ok** [comment]
If you open the game from the emulator's menu, does it run at normal speed? Does it run at the same speed on other emulators?

**Comment by zartan9** [comment]
hi i running it from
https://c64online.com/c64-online-emulator/ its slow (says 18 on bottom, the speed?)
and vice same thing. I tried it from the batch and its really quick! if you know another emulator i can try  it... thanks

**Comment by haroldo-ok** [comment]
Weird; I guess the rendering routine is slow, and would probably be slow on real hardware.. but the interesting part is that it is running differently on the emulator's warp mode depending on how it is loaded.

**Comment by zartan9** [comment]
yes. I wish i could help you speed up your projects (pun intended!) to keep them updated (but I dont know code) if you need any graphic stuff or easy to do tasks
Id be happy to help.

**Comment by haroldo-ok** [comment]
Okay, registered an issue to optimize the screen rendering later:
https://github.com/haroldo-ok/BlocklyVN8bit/issues/18
